### PR TITLE
Jmoses/update contributing doc

### LINKED ISSUE #4991: Use consistent sentence structure and adjust wording in CONTRIBUTING.md
In `CONTRIBUTING.md` we could possibly be more consistent around sentence structure and bullet points by not having bullet points and non complete sentences end in `.` (periods). 

Separately under the `Issue and Pull Request Labels` section, the description for `reviewer-needs-to-reproduce` currently reads: `Likely bugs, but haven't been reliably reproduced by a reviewer.`.

I'm suggesting changing the description to `Possible bugs, that haven't been reliably reproduced by a reviewer, yet` or `Possible bugs, that haven't yet been reliably reproduced by a reviewer` (thoughts/suggestions welcomed). This change helps convey the message that while a reviewer has not been able to reproduce it at this time, it is still on the radar. Plus I believe it reads a little easier.

PR Incoming, please do what you wish with it :)

### PR DESCRIPTION
**Pull request by JonahMoses**
- removes `.` to be consistent with sentence structure (should not have `.` if not a full sentence or as a bullet point item).
- adjusts wording for `reviewer-needs-to-reproduce` description text to be a little friendlier. 

❤️ Closes #4991

### CODE REVIEW

**Comment by iAmWillShepherd** [comment]
Going to leave this open until tomorrow in case anyone from @desktop/core wants to chime in.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -149,27 +149,27 @@ pull requests.
...
-| `Windows` | [search](https://github.com/desktop/desktop/labels/Windows)  | Issues specific to Windows users. |
-| `codemirror` | [search](https://github.com/desktop/desktop/labels/codemirror)  | Issues related to our use of [CodeMirror](https://codemirror.net/) that may require upstream fixes. |
-| `electron` | [search](https://github.com/desktop/desktop/labels/electron) | Issues related to our use of [Electron](https://electron.atom.io) that may require upstream fixes. |
+| `enhancement` | [search](https://github.com/desktop/desktop/labels/enhancement) | Feature requests |
+| `bug` | [search](https://github.com/desktop/desktop/labels/bug)  | Confirmed bugs or reports that are very likely to be bugs |
+| `more-information-needed` | [search](https://github.com/desktop/desktop/labels/more-information-needed) | More information needs to be collected about these problems or feature requests (e.g. steps to reproduce) |
+| `reviewer-needs-to-reproduce` | [search](https://github.com/desktop/desktop/labels/reviewer-needs-to-reproduce)  | Possible bugs, that haven't been reliably reproduced by a reviewer, yet |
```
Comment (shiftkey) [inline .github/CONTRIBUTING.md]: I like this wording, but the `yet` feels like strange at the end. What about

> Potential bugs which need to be reliably reproduced by a reviewer

Comment (JonahMoses) [reply .github/CONTRIBUTING.md]: Yes! I also felt that way around the trailing yet. 

Maybe:
`Potential bugs that still need to be reliably reproduced by a reviewer`?

One part of the message that I was hoping to convey is an "in progress" feeling. By including a "yet" or "still" to the label description, to me, represents this concept that it just hasn't not been gotten to at this time, but is recognized and will be addressed eventually. Thoughts?

**Comment by JonahMoses** [comment]
@shiftkey / @iAmWillShepherd Proposed updates committed onto a wip commit then squashed onto original `reviewer-needs-to-reproduce label description` commit  then `-f` pushed, which looks to have removed the original approval (rightly so).

**Comment by JonahMoses** [comment]
Thanks @iAmWillShepherd & @shiftkey 
🎉 ❣️

**Pull request merged**